docs/source/top_level_elements/genre.rst

```diff
@@ -0,0 +1,163 @@
+genre
+=====
+
+About
+-----
+
+This section describes the different values for `genre` that we currently have in our Islandora repository.
+
+genre[@authority = 'dct']
+-------------------------
+
+We will explore other options for records using `@authority = 'dct'` or uncontrolled vocabularies, e.g. mapping values as `typeOfResource`. Many of the records in Volunteer Voices have multiple `genre`s; e.g. `volvoices:14311 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:14311/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="dct">text</genre>
+    <genre>letter</genre>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:type "text" ;
```

Comment:
Sorry I didn't catch this previously, but is it possible for us to use URIs for genre[@authority="dct"]/dcterms:type? Islandora has outlined the values in their mapping. Also, technically the capitalization is different in dcterms versus MODS typeOfResource, so "text" in MODS is "Text" in dcterms. Ultimately though it would be best if we could write in a URI for dcterms. This would make it consistent with the other values from MODS typeOfResource.

Reply:
@mlhale7 thanks for pointing this out. I think we may have talked about this and I overlooked making this change.

Reply:
@mlhale7 I updated this in d77bff1a - let me know if it doesn't resolve this comment.

Reply:
Thanks - this is great!